mantidproject/mantid
# IN16B first tube Q

**Description of work.**
The analysers in front of the first tube of IN16B are not focused on the sample position, but on the midpoint of the sample and the tube center. Because of this there is no unique neutronic image for most of the pixels. This PR adds an IDF that is tweaked to work around this problem by squeezing the tube into a point in its center in the neutronic representation. Also does some cleanup in test files. IDFs are generated by PR https://github.com/mantidproject/mantidgeometry/pull/163 in geometry.

**Report to:** [ILL/Appel]

**To test:**
Use the `136558.nxs` run in ILL/IN16B under unit test data.
```
Load(Filename='136558', OutputWorkspace='in16bf') 
GroupDetectors(InputWorkspace='in16bf', OutputWorkspace='grf', MapFile='%PATH_TO_MANTID_SOURCE%/instrument/Grouping/IN16B_Grouping.xml') 
ConvertSpectrumAxis(InputWorkspace='grf', OutputWorkspace='qf', Target='ElasticQ', EFixed=2.0812, OrderAxis=False)
print(mtd['qf'].getAxis(1).extractValues()[0])
```
Should be `~0.4356` \AA.
Also click the little arrow next to the loaded workspace and confirm that it loads the IN16BF_Definition.xml.



Part of #27148 . 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
